config/config.go

```diff
@@ -774,20 +774,16 @@ func (c *Config) ToOptions() skipper.Options {
 		SuppressRouteUpdateLogs:             c.SuppressRouteUpdateLogs,
 
 		// route sources:
-		EtcdUrls:                  eus,
-		EtcdPrefix:                c.EtcdPrefix,
-		EtcdWaitTimeout:           c.EtcdTimeout,
-		EtcdInsecure:              c.EtcdInsecure,
-		EtcdOAuthToken:            c.EtcdOAuthToken,
-		EtcdUsername:              c.EtcdUsername,
-		EtcdPassword:              c.EtcdPassword,
-		InnkeeperUrl:              c.InnkeeperURL,
-		InnkeeperAuthToken:        c.InnkeeperAuthToken,
-		InnkeeperPreRouteFilters:  c.InnkeeperPreRouteFilters,
-		InnkeeperPostRouteFilters: c.InnkeeperPostRouteFilters,
```

Comment:
I thin there are flags for these that still need to be removed

Reply:
Yes, thanks some todo

Reply:
done